include/wx/textctrl.h

```diff
@@ -756,6 +756,12 @@ class WXDLLIMPEXP_CORE wxTextCtrlBase : public wxControl,
 
     virtual const wxTextEntry* WXGetTextEntry() const wxOVERRIDE { return this; }
 
+#if wxUSE_SPELLCHECK
+    // Use native spelling and grammar checking functions.
+    virtual bool EnableSpellCheck(bool enable = true) { (void)enable; return false; }
```

Comment:
Just for style consistency:
```suggestion
    virtual bool EnableSpellCheck(bool WXUNUSED(enable) = true) { return false; }
```

Reply:
Sorry, I forgot that WXUNUSED went in the declaration - I was trying it in the implementation which of course didn't work!